tests/com/sun/cldc/i18n/UTF_8_Writer_sizeOf.java

```diff
@@ -0,0 +1,35 @@
+package com.sun.cldc.i18n;
+
+import gnu.testlet.Testlet;
+import gnu.testlet.TestHarness;
+import com.sun.cldc.i18n.j2me.UTF_8_Writer;
+import com.sun.cldc.i18n.Helper;
+import com.sun.cldc.i18n.StreamWriter;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class UTF_8_Writer_sizeOf implements Testlet {
+  public int getExpectedPass() { return 1; }
+  public int getExpectedFail() { return 0; }
+  public int getExpectedKnownFail() { return 0; }
+
+  public void test (TestHarness th)
+  {
+    try {
+        String str = "abcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZabcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZ";
```

Comment:
No, I just copied the "skeleton" from another file, and this caused some problems with the indentation.